app/workers/vendor_integration_stats_worker.rb

```
@@ -23,11 +23,11 @@ def perform(vendor_name)
 private
 
   def fetch_webhook_url
-    ENV.fetch "#{@vendor_name.upcase}_INTEGRATION_STATS_SLACK_URL"
+    ENV.fetch("#{@vendor_name.upcase}_INTEGRATION_STATS_SLACK_URL", nil)
```

should we raise an error if the vendor isn't found just to be alerted?

---

as in if we have a typo with the vendor name?